### PR TITLE
Added several package versions and a yum repo file

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: build/vanagon/components/ruby-2.6.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   pkg.environment 'optflags', '-O2'
 
-  special_flags = " --prefix=#{ruby_dir} --with-opt-dir=#{settings[:prefix]} --enable-dtrace --with-baseruby=#{host_ruby}"
```
Comment (dhollinger) [inline build/vanagon/components/ruby-2.6.rb]: DTrace is old and, in the case of Debian, not included in the system at all. Not sure if it's really needed for this anyway.

File: README.md [hunk 2]
```diff
@@ -31,22 +31,31 @@ Please see the pre-2.x [documentation](docs/LEGACY.md) if using the old gem-base
...
-This assumes you already have the correct Ruby version installed in the system or via a tool like `rbenv`, `rvm`, or `asdf`.
+#### Ubuntu/Debian
 
-1. Download the latest release from the [Releases](https://github.com/voxpupuli/puppet_webhook/releases) page.
-2. Extract the tarball into the desired directory:
+Add the Voxpupuli GPG key and repository:
 ```bash
-$ tar -xzvf puppet_webhook-latest.tar.gz -C /opt/puppet_webhook
+wget -qO - "https://bintray.com/user/downloadSubjectPublicKey?username=voxpupuli" | sudo apt-key add -
+echo "deb https://dl.bintray.com/voxpupuli/deb $(grep "VERSION_CODENAME=" /etc/os-release |awk -F= '{print $2}')  main" | sudo tee -a /etc/apt/sources.list.d/voxpupuli.list
```
Comment (ekohl) [inline README.md]:
```suggestion
echo "deb https://dl.bintray.com/voxpupuli/deb $(awk -F= '/VERSION_CODENAME=/ {print $2}' /etc/os-release) main" | sudo tee -a /etc/apt/sources.list.d/voxpupuli.list
```

File: build/vanagon/platforms/el-7-x86_64.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  packages = %w[
+    bzip2-devel
+    readline-devel
+    make
+    pkgconfig
+    zlib-devel
+    pl-cmake
+    pl-gcc
+    systemtap-sdt-devel
+    java-1.8.0-openjdk-headless
```
Comment (ekohl) [inline build/vanagon/platforms/el-7-x86_64.rb]: Why is java needed?

File: build/vanagon/platforms/el-7-x86_64.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  ]
+
+  plat.provision_with "yum install --assumeyes #{packages.join(' ')}"
+
+  plat.output_dir File.join('el', 'PC1')
```
Comment (ekohl) [inline build/vanagon/platforms/el-7-x86_64.rb]: PC1 looks like the old Puppet 4 configs

File: build/vanagon/platforms/el-7-x86_64.rb (hunk 3 above)
Comment (dhollinger) [reply build/vanagon/platforms/el-7-x86_64.rb]: This (and the below) are both leftovers from when I copied the files from the vanagon puppet_runtime repo. Planned cleanup as part of the next few releases.

File: build/vanagon/platforms/el-7-x86_64.rb (hunk 4 above)
Comment (dhollinger) [reply build/vanagon/platforms/el-7-x86_64.rb]: see above ^

**Pull request merged**